sdk/provisioning/Azure.Provisioning.Storage/src/Generated/BlobContainer.cs

```diff
@@ -280,6 +280,7 @@ public BlobContainer(string bicepIdentifier, string? resourceVersion = default)
     /// </summary>
     protected override void DefineProvisionableProperties()
     {
+        base.DefineProvisionableProperties();
```

Comment:
this is caused by a recent generator change.
The `DefineProvisionableProperties()` in base type does nothing therefore adding this line does not change anything as a result.